R/bundle.R

```diff
@@ -164,7 +164,12 @@ bundleApp <- function(appName, appDir, appFiles, appPrimaryDoc, assetTypeName,
                       contentCategory) {
 
   # infer the mode of the application from its layout
-  appMode <- inferAppMode(appDir, appPrimaryDoc, appFiles)
+  # unless we're an API, in which case, we're API mode.
+  if (contentCategory == "api") {
+    appMode <- "api"
+  } else {
+    appMode <- inferAppMode(appDir, appPrimaryDoc, appFiles)
```

Comment:
Should `inferAppMode` detect the `api` content type? (We don't generally require users to specify `contentCategory` unless the situation is truly ambiguous.)